avocado/utils/configure_network.py

```diff
@@ -201,3 +202,18 @@ def is_interface_link_up(interface):
         log.info("Interface %s link is up", interface)
         return True
     return False
+
+
+def get_interface_ip(interface):
+    """
+    Get the IP address from a network interface
+    :param interface: name of the interface
+    :return: List of IP addresses.
+    """
+    cmd = "ip addr show %s" % interface
+    for line in process.system_output(cmd, shell=True).decode("utf-8") \
+                                                       .split('\n'):
+        match = re.findall( r'^\s+inet ([0-9]+(?:\.[0-9]+){3}).*', line)
+        if match:
+            break
+    return match
```

Comment:
`match` is defined only inside the `for`, so if there is no output to run on this for, this code will be broken, right?

Reply:
@beraldoleal Thanks :)  
I've recheck it with the next cases:
1 Incorrect interface name: got error about it
2 Interface without provided IP address: got an empty list

The 'cmd' is defined command, the loop will be run in any case even with incorrect data.
But I'll improve it maybe with log.error()